### ISSUE TITLE
`prepare_for_training` does not work for multi-label dataset

### ISSUE THREAD

**Issue by davidberenstein1957** [recognai/rubrix#2132]
**Describe the bug**
I cannot use multi-label dataset.`prepare_for_training` directly.

**To Reproduce**
1. Go to any multi-label dataset.
2. export the dataset via `prepare_for_training`
3. use for training directly

**Expected behavior**
multi-label datasets ought to be delivered with `https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.MultiLabelBinarizer.html` or they cannot be used for training

**Screenshots**
N.A.

**Environment (please complete the following information):**
 - OS [e.g. iOS]: N.A.
 - Browser [e.g. chrome, safari]: N.A.
 - Argilla Version [e.g. 1.0.0]:1.1.1
 - ElasticSearch Version [e.g. 7.10.2]:N.A.
 - Docker Image (optional) [e.g. argilla:v1.0.0]:N.A.

**Additional context**
N.A.
